opsgenie/resource_opsgenie_team_membership.go

```diff
@@ -0,0 +1,159 @@
+package opsgenie
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/opsgenie/opsgenie-go-sdk-v2/team"
+	"log"
+	"strings"
+)
+
+func resourceOpsGenieTeamMembership() *schema.Resource { //TODO encode the e-mail addrs (e.g. because of +)? https://github.com/opsgenie/opsgenie-go-sdk-v2/issues/62
+	return &schema.Resource{
+		Create: resourceOpsGenieTeamMembershipCreate,
+		Read:   handleNonExistentResource(resourceOpsGenieTeamMembershipRead),
+		//Update: resourceOpsGenieTeamMembershipUpdate, // requires https://github.com/opsgenie/opsgenie-go-sdk-v2/issues/59
```

Comment:
pr looks good i'll talk with the internal team about go-sdk-v2 membership update status
Thank you @arnisoph

Reply:
we don't have update support. if we introduce this users cannot change roles in their teams. I'll create a internal ticket and let you know. 

maybe we can use update function to 
-> get team -> update our user role -> update team flow 
This way we can update our team members ?

Reply:
I don't even plan to implement it, it's just a comment :D